pkg/clients/signingclienttracker.go

```diff
@@ -1,29 +1,26 @@
-/*
-Copyright IBM Corp. All Rights Reserved.
-
-SPDX-License-Identifier: Apache-2.0
-*/
-
```

Comment:
Keep it?

Reply:
The content of this file changed almost completely since IBM, I think here we can remove it without problems.

Reply:
https://github.com/filecoin-project/mir/compare/10658fbe24ca2bb33ac3151daf7d6f610f4a16e2...matejpavlovic:general-modules#diff-c9a08836d7ab42d845d17eddc4a99521169a6c945cb85923a8f774c1e739c85d

Reply:
Hm I thought I wrote it much more recently. Fair enough, let's keep the notice in this one.